.github/workflows/build.yml

```diff
@@ -88,6 +88,24 @@ jobs:
           mv release/windows/juno.exe release/windows/juno-${{steps.version_check.outputs.releaseVersion}}-windows.exe
           mv release/macos/juno release/macos/juno-${{steps.version_check.outputs.releaseVersion}}-macos
           mv release/linux/juno release/linux/juno-${{steps.version_check.outputs.releaseVersion}}-linux
+        
+      - name: Publish ubuntu docker image
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: bytesonus/juno
+          tags: latest,${{ steps.version_check.outputs.releaseTags }}
+          dockerfile: ./Dockerfile
+        
+      - name: Publish alpine docker image
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: bytesonus/juno
+          tags: latest-alpine,${{ steps.version_check.outputs.rawVersion }}-alpine
```

Comment:
should probably have the same behaviour for alpine and ubuntu image versioning. I'll see how we can do that using version-check